packages/riipen-ui/src/components/Tooltip.jsx

```diff
@@ -406,97 +311,66 @@ class Tooltip extends React.Component {
    * @param {string} value - value to compare
    * @returns {string} Opposing direciton, or just value if directions do not oppose.
    */
-  getOppositeDirection = (direction1, direction2, value) => {
-    if (value === direction1) {
-      return direction2;
-    } else if (value === direction2) {
-      return direction1;
+  const getOppositeDirection = (direction, oppositeDirection, value) => {
+    if (value === direction) {
+      return oppositeDirection;
+    } else if (value === oppositeDirection) {
+      return direction;
     }
     return value;
   };
 
-  blur = () => {
+  const blur = () => {
```

Comment:
I think if there is a focus error again, this is the perpetrator. But it seems okay as a function component?